tests/dragonfly/connection_test.py

```diff
@@ -287,6 +287,42 @@ def generate(max):
     assert state, message
 
 
+"""
+Test PUBSUB NUMSUB command.
+"""
+
+
+@pytest.mark.asyncio
+async def test_pubsub_subcommand_for_numsub(async_client):
+    subs1 = [async_client.pubsub() for i in range(5)]
+    for s in subs1:
+        await s.subscribe("channel_name1")
+    result = await async_client.pubsub_numsub("channel_name1")
+
+    for s in subs1:
+        await s.unsubscribe("channel_name1")
+    assert result[0][0] == "channel_name1" and result[0][1] == 5
```

Comment:
I'd move this assert up to be after line 300
Additionally, could you also add another assert here that `channel_name1` has `0` subs?
(while at it, could you also try a non-existing channel name?)

Reply:
If you'd like to do that as a follow-up PR - I'd be happy to review it :)

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/78039c224abd514ff0f8e0504aab5c4787a564f3